.tekton/single-arch-build-pipeline.yaml

```
@@ -82,7 +63,7 @@ spec:
       description: Image tag expiration time, time values could be something like
         1h, 2d, 3w for hours, days, and weeks, respectively.
       name: image-expires-after
-    - default: "true"
+    - default: "false"
```

```suggestion
    - default: "true"
```
The last few builds did not require source image to be built, now EC validation is failing since its missing. Please enable this.

---

Updated.